src/containers/Tenant/Info/ExternalTable/ExternalTable.tsx

```diff
@@ -15,27 +15,30 @@ import './ExternalTable.scss';
 
 const b = cn('ydb-external-table-info');
 
-const prepareExternalTableSummary = (
-    data: TEvDescribeSchemeResult,
-    pathToDataSource: string,
-): InfoViewerItem[] => {
+const prepareExternalTableSummary = (data: TEvDescribeSchemeResult, pathToDataSource: string) => {
     const {CreateStep} = data.PathDescription?.Self || {};
     const {SourceType, DataSourcePath} = data.PathDescription?.ExternalTableDescription || {};
 
     const dataSourceName = DataSourcePath?.split('/').pop();
 
-    return [
+    const info: InfoViewerItem[] = [
         {label: i18n('external-objects.source-type'), value: SourceType},
-        formatCommonItem('CreateStep', CreateStep),
-        {
-            label: i18n('external-objects.data-source'),
-            value: DataSourcePath && (
-                <span title={DataSourcePath}>
-                    <LinkWithIcon title={dataSourceName || ''} url={pathToDataSource} />
-                </span>
-            ),
-        },
     ];
+
+    if (Number(CreateStep)) {
```

Comment:
Mb it would be more consistent to display EMPTY_DATA_PLACEHOLDER (via date formatter and defaultValue) everywhere?

Reply:
I don't think so. Normally, we have 0 `CreateStep` for certain list of entities - Domain, databases, system tables. So in your case there will be always placeholder there. I added placeholder in `formatCommonItem` just as fallback, but you are right - it's better to check correctness inside `formatDateTime`